src/clj/game/cards-ice.clj

```diff
@@ -336,9 +337,10 @@
                                       (->> (vec (.split (:subtype card) " - "))
                                            (concat [target])
                                            (join " - "))))
+                      (update! (assoc card :subtype-target target))
```

Comment:
Not much to add. Yes, be careful with multiple `update!` to the same card. Values are immutable so the first does not change your local copy of the card data.